### DISCUSSION TITLE
Ability to sync multiple provider accounts to single jellyfin user

### DISCUSSION THREAD

**Discussion by lichenophile** [vosmiic/jellyfin-ani-sync#129] [Ideas]
My house uses a shared jf user when we watch media together, is it possible to have this plugin sync multiple anilist/etc users when our 'group' jf user watches an episode?

At present, authorising a users list for syncing using an already authed jf user seems to override the pre-existing user list sync.

**Comment by vosmiic** [comment]
This is a cool idea and I believe possible, I'll take a look into implementing this when I get a chance
